### PR TITLE
Fixes error for merchants not yet migrated to v2 amazon keys

### PR DESCRIPTION
**Pull request by dpanta94**
### All Submissions:

* [x] Does your code follow the [Extendables](https://extendomattic.wordpress.com/standardizations/) standards?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changes proposed in this Pull Request:

We won't load the Express Gateway for merchants not yet migrated to V2 Amazon Keys and the FE is going to reflect that.

Closes woo#225 .

### How to test the changes in this Pull Request:

1. While using V2 keys of Amazon and WooCommerce Blocks, Amazon Express should appear in the cart and checkout blocks.
2. While using V1 keys of Amazon and WooCommerce Blocks, Amazon Express shoud not appear in the cart and checkout blocks.
3. WHile using V1 keys of Amazon and WooCommerce Blocks, classic gateway should not be present.
4. While using V1 keys of Amazon and WooCommerce Blocks, no error should be thrown.

### Other information:

* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changelog entry

> Fix - Error for merchants still using V1 Amazon Keys

### CODE REVIEW

**Review by gen4ik-saucal — APPROVED** [review]
LGTM!

**Review by gen4ik-saucal — APPROVED** [review]
:green_apple:

File: includes/compats/woo-blocks/class-wc-amazon-payments-advanced-block-compat-express.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ class WC_Amazon_Payments_Advanced_Block_Compat_Express extends WC_Amazon_Payment
 	 * @return boolean
 	 */
 	public function is_active() {
-		return wc_apa()->get_gateway()->is_available();
+		return wc_apa()->should_express_be_loaded() && wc_apa()->get_gateway()->is_available();;
```
Comment (gen4ik-saucal) [inline includes/compats/woo-blocks/class-wc-amazon-payments-advanced-block-compat-express.php:33]: ;; needed here?

**Review by gen4ik-saucal — APPROVED** [review]
Goot to go. Just remove double ';' if it is not needed.

File: includes/compats/woo-blocks/class-wc-amazon-payments-advanced-block-compat-express.php (hunk 1 above)
Comment (dpanta94) [reply includes/compats/woo-blocks/class-wc-amazon-payments-advanced-block-compat-express.php]: removed, thank you!

**Comment by dpanta94** [comment]
@gen4ik-saucal there is one more commit here. Please check it when possible, thank you!

**Review by gen4ik-saucal — APPROVED** [review]
LGTM

**Comment by dpanta94** [comment]
Opened PR in woo repo pull#240

**Pull request merged**